next-level/skills/execute/SKILL.md

```
@@ -0,0 +1,103 @@
+   ```bash
+   gh issue list --milestone "<epic-milestone>" --state open --json number,title,body,labels --limit 50
+   ```
+3. **Parse dependencies**: Read each issue body for "Blocked by: #N" references
+4. **Build execution order**: Topological sort respecting dependencies
+5. **Show plan**: Present the execution order to the user for confirmation
+
+## Phase 2: Execute Tasks
+
+For each task in execution order:
+
+### 2a. Start Task
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)